content/docs/command-reference/status.md

```diff
@@ -57,7 +57,7 @@ description_, as detailed below:
 
 - _always changed_ means that this is a `.dvc` file with no dependencies (see
   `dvc add`) or that the stage in `dvc.yaml` has the `always_changed: true`
-  value set (see `--always-changed` option in `dvc run`).
+  value set (see `--always-changed` option in `dvc stage add`).
```

Comment:
Actually these can now link directly to the option anchor like this 🙂 
```suggestion
  value set (see `dvc stage add --always-changed`).
```
> Per https://github.com/iterative/dvc.org/pull/3140

But we prob. need a separate issue to update these

Reply:
Extracted to https://github.com/iterative/dvc.org/issues/3236